### ISSUE TITLE
Path is not rendered correctly

### ISSUE THREAD

**Issue by jry2** [memononen/nanosvg#39]
I'm not sure if this is a bug, but following path is not rendered correctly:

```
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" id="Report" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="492px" height="485px" viewBox="0 0 492 485" enable-background="new 0 0 492 485" xml:space="preserve">
<path fill-rule="evenodd" clip-rule="evenodd" fill="#ABABAB" d="M188.603,167.706c80.044,0,144.958,64.914,144.958,144.959
	c0,80.044-64.914,144.958-144.958,144.958c-80.046,0-144.958-64.914-144.958-144.958
	C43.646,232.619,108.558,167.706,188.603,167.706L188.603,167.706z M185.009,172.391c75.474,0,136.679,61.207,136.679,136.683
	c0,75.466-61.205,136.673-136.679,136.673c-75.472,0-136.679-61.207-136.679-136.673C48.33,233.598,109.537,172.391,185.009,172.391
	L185.009,172.391z"/>
</svg>
```
There should be transparent center part of image that is filled when rendered with Nano SVG.

**Comment by memononen** [comment]
Should be fixed now, added even-odd fillrule support 5966d6e77a439a181348f9412d0f719f2a6f8552

**Comment by jry2** [comment]
Problem is fixed, thank you.
